cfme/infrastructure/virtual_machines.py

```diff
@@ -155,7 +155,7 @@ class VmsTemplatesAllView(InfraVmView):
     def is_displayed(self):
         return (
             self.in_infra_vms and
-            self.sidebar.vmstemplates.tree.currently_selected == 'All VMs & Templates' and
+            self.sidebar.vmstemplates.tree.currently_selected[0] == 'All VMs & Templates' and
```

Comment:
can you please explain what is it for ?

Reply:
@izapolsk Here when I checked self.sidebar.vmstemplates.tree.currently_selected. it output is a list of nodes of the path of the accordion tree.

1)Case 1:  'All VMs & Templates' is selected 
output: [u'All VMs & Templates']

2)Case 2: 'vSphere 5.5 (nested)' is selected
Output: [u'All VMs & Templates', u'vSphere 5.5 (nested)']

So we need only the first element of the list to check correct accordion tab is selected or not.